curriculum/js1/9.js

```diff
@@ -4,11 +4,11 @@
  * @returns {number}
  */
 
-const solution = (a, b = (a%7)) => {
-  if (b===0) {
+const solution = (a) => {
+  if (a % 7) === 0) {
```

Comment:
Did you test this solution? There's a syntax error on this line (hint: brackets/parentheses). The test should fail.